themes/app/templates/Layout/Page.ss

```diff
@@ -1,4 +1,13 @@
-<h2>$Title</h2>
-$Content
-$Form
-$ElementalArea
+$Breadcrumbs
+<div class="container">
+    <div class="row">
+        <div class="typography col-lg-8">
```

Comment:
You can use `col-md-8` here

Reply:
I purposefully set as lg because I want it to be full width if on tablet or smaller devices.

Reply:
ahh I see, alrighty 👍